test/test_linting.jl

```diff
@@ -1,17 +1,12 @@
 using PlutoTeachingTools
 using Test
 
-using JuliaFormatter: JuliaFormatter
 using Aqua: Aqua
 using JET: JET
 using ExplicitImports: ExplicitImports
 
-@testset "Code formatting" begin
-    @test JuliaFormatter.format(PlutoTeachingTools; verbose=false, overwrite=false)
-end
-
 @testset "Aqua tests" begin
-    Aqua.test_all(PlutoTeachingTools; ambiguities=false)
+    Aqua.test_all(PlutoTeachingTools; ambiguities = false)
```

Comment:
```suggestion
    Aqua.test_all(PlutoTeachingTools; ambiguities=false)
```

nevermind - the action is already switched off in this PR